app/main.py

```diff
@@ -12,7 +13,7 @@ def __init__(
         self.hidden = hidden
         Animal.alive.append(self)
 
-    def is_alive(self) -> None:
+    def keep_alive(self) -> None:
```

Comment:
keep alive makes animal dead(removes from list)? xD
sry, but you need to try more )